datahub-graphql-core/src/main/java/com/linkedin/datahub/graphql/types/ingestion/IngestionSourceMapper.java

```diff
@@ -0,0 +1,24 @@
+package com.linkedin.datahub.graphql.types.ingestion;
+
+import com.linkedin.datahub.graphql.QueryContext;
+import com.linkedin.datahub.graphql.generated.IngestionSource;
+import com.linkedin.datahub.graphql.resolvers.ingest.IngestionResolverUtils;
+import com.linkedin.datahub.graphql.types.mappers.ModelMapper;
+import com.linkedin.entity.EntityResponse;
+
+import javax.annotation.Nonnull;
+import javax.annotation.Nullable;
+
+public class IngestionSourceMapper implements ModelMapper<EntityResponse, IngestionSource> {
+    public static final IngestionSourceMapper INSTANCE = new IngestionSourceMapper();
+
+    public static IngestionSource map(
+            @Nullable QueryContext context, @Nonnull final EntityResponse entityResponse) {
+        return INSTANCE.apply(context, entityResponse);
+    }
+
+    @Override
+    public IngestionSource apply(@Nullable QueryContext context, @Nonnull EntityResponse input) {
+        return IngestionResolverUtils.mapIngestionSource(input);
```

Comment:
ideally we just have this one `IngestionSourceMapper` instead of having the mapping in `IngestionResolverUtils` - so if we can pull that into here and use this mapper everywhere to be consistent with other entities that would be good